swagger.json

```diff
@@ -147,12 +147,12 @@
                   "email": {
                     "type": "string",
                     "format": "email",
-                    "example": "homer.simpson@sfpp.com"
+                    "example": "homer2.simpson@sfpp.com"
```

Comment:
I believe the data seeder creates user "homer.simpson@sfpp.com" and think it would be better to leave it as the example so that it works out of the box.  If I'm missing something, let me know.

Reply:
Whoops, I didn't notice that this was in POST /User.  My bad. Ignore.